docs/watchman.md

```diff
@@ -1,12 +1,12 @@
-[Moov Watchman](https://github.com/moov-io/watchman) is used by Customers to compare Customer information against Anit-Money Laundering (AML) and sanction lists provided by various government agencies. The primary list checked is OFAC as a component of Know Your Customer (KYC). These checks are required of all US businesses.
+[Moov Watchman](https://github.com/moov-io/watchman) is used by Customers to compare customer information against Anit-Money Laundering (AML) and sanction lists provided by various government agencies. The primary list is checked by Office of Foreign Asset Control (OFAC) as a component of Know Your Customer (KYC). These checks are required of all US businesses.
```

Comment:
This new version isn't quite accurate. The list isn't checked _by_ OFAC, the list checked is commonly referred to as OFAC, after the government office responsible for producing it. The OFAC lists (like the SDN, for example) are the primary sanctions/blacklist sources.

Reply:
Could we say `the primary list is matched against OFAC`?

Reply:
No I don't think so. It was worded that way before because the OFAC lists _are_ the "primary list". So a single customer or entity name is what's being checked against the OFAC lists, being referred to collectively as the primary list.

Reply:
Ah I see, I'll change it back to the original wording.

Reply:
The list is managed by the US Treasury. It's called the "OFAC list", so Watchman downloads, indexes and offers a search API over the "OFAC list". 

Their search page: https://sanctionssearch.ofac.treas.gov/